lifecycle-application/src/test/java/io/quarkus/ts/openshift/lifecycle/LifecycleApplicationOpenShiftIT.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.ts.openshift.lifecycle;
+
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+public class LifecycleApplicationOpenShiftIT {
+
+    private static final String LOGGING_PROPERTY = "-Djava.util.logging.manager=org.jboss.logmanager.LogManager";
+    private static final String RUNTIME_LABEL = "app.openshift.io/runtime";
+    private static final String RUNTIME_QUARKUS = "quarkus";
+
+    @TestResource
+    private OpenShiftClient openshiftClient;
+
+    @Test
+    public void shouldArgumentsNotContainLoggingProperty() {
+        when().get("/args")
+        .then().statusCode(200)
+                .body(not(containsString(LOGGING_PROPERTY)));
+    }
+
+    @Test
+    public void shouldPrintMessagesFromQuarkusMain() {
+        PodList pods = openshiftClient.pods().withLabel(RUNTIME_LABEL, RUNTIME_QUARKUS).list();
+        assertEquals(1, pods.getItems().size(), "Found more than one pod with Quarkus runtime");
+
+        String actualLog = openshiftClient.pods().withName(pods.getItems().get(0).getMetadata().getName()).getLog();
+
+        Optional<String> argumentsLine = actualLog.lines().filter(line -> line.contains("Received arguments:")).findFirst();
```

Comment:
`actualLog.lines()` is JDK 11 API and the TS is set to 1.8.

We run it with Java 11, not so sure if we should switch to 11 just because this one line.
If there are more places that could benefit from Java 11 APIs, we should do the switch.

Reply:
`"Received arguments:"` could be a constant defined as private static final String

Reply:
Changed